tools/filters/sff_extract.py

```diff
@@ -47,7 +47,7 @@
 linkerlengths = {}  # type: ignore
 
 # set to true if something really fishy is going on with the sequences
-stern_warning = True
+stern_warning = False
```

Comment:
Looks innocent enough, but do we really want to apply these "fixes" to legacy code we aren't familiar with ?

Reply:
@mvdbeek I was reluctant to fix this as well, but it was so clearly wrong (since the first commit of this code). I did some testing before pushing this using https://github.com/galaxyproject/galaxy-test-data/blob/master/2.sff and some other test data.
Before the changes, `stern_warning` was always True, so the `foundproblem` string was always printed to the tool stdout and the script was always exiting with status 1, but this is ignored by Galaxy because it's a legacy tool with no profile specified.
With these changes, the message is printed on the tool stdout a bit less often, but there is no other functional change since the exit code is still ignored by Galaxy.